### ISSUE TITLE
Find a group of users with experience on Hospital acquired infections

### ISSUE THREAD

**Issue by genterist** [genterist/whiteWolf#3]

**Comment by genterist** [comment]
Team,
A email was sent to Captain M at USPHS (check your inbox for CC'ed version). I will have a discussion with her later on today regarding how she can help us find survey participants, and project mentors (health care managers). I think we also need to look into other options to push our survey to more people (Dr. Menzes mentioned 2 in class). 
--------

**Comment by genterist** [comment]
@mfarve1 and @ChristineTzeng 

I've just took a glimpse at MTurk and not sure if they have an option to select just the people working in healthcare. I did a little research and found out that Pollfish.com has that option and cheap (100 responses for a price of $160)
![screen shot 2017-01-19 at 11 14 43 pm](https://cloud.githubusercontent.com/assets/21042389/22135616/44479ec6-de9d-11e6-969b-6412c39ca606.png)

The down side is that we have to buy a minimum of 100 responses. This is just one option, alternative to Mturk (remember Jack says Mturk may takes up to $4 per response and there is an extra fee for audience targeting if i am not mistaken).

HOWEVER, this paper shows MTurk can be really cheap yet effective
[https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013244/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5013244/)
